src/sgame/sg_bot.cpp

```diff
@@ -263,6 +262,188 @@ bool G_BotSetDefaults( int clientNum, team_t team, int skill, Str::StringRef beh
 	return true;
 }
 
+struct botSkillTreeElement_t {
```

Comment:
Honestly, I would like to see the skill tree stuff moved to a separate file. It's conceptually different enough to merit it I think.

Reply:
Makes sense

Reply:
Did that in a new commit to avoid having to do that for every commit.